### PR TITLE
[stable10] Preliminary changelog 10.0.9

### PR DESCRIPTION
**Pull request by PVince81**
Changelog with items that were merged after 10.0.8

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+- Subadmin shouldn't be able to add users to their groups via API - [#31337](https://github.com/owncloud/core/issues/31337)
+- Remove session token creation for dav requests during loadApps - [#31428](https://github.com/owncloud/core/issues/31428)
+- Fix overflowing public share names in the share panel - [#31369](https://github.com/owncloud/core/issues/31369)
+- Fix occ user:sync to sync quota from preferences after upgrade if backend provided no quota - [#31360](https://github.com/owncloud/core/issues/31360)
+- Fix for Redis dev editions - [#31282](https://github.com/owncloud/core/issues/31282)
+- Fix mail debug message recipient field - [#31227](https://github.com/owncloud/core/issues/31227)
+- Prevent infinite loop in case of error in "log" event handler - [#31247](https://github.com/owncloud/core/issues/31247)
+- Convert FileContentNotAllowedException into DAVForbiddenException to … - [#31260](https://github.com/owncloud/core/issues/31260)
+- Add back robots.txt in the release - [#31248](https://github.com/owncloud/core/issues/31248)
+- Fix extractRequestTokenFromResponse method name - [#31217](https://github.com/owncloud/core/issues/31217)
```
Comment (phil-davis) [inline CHANGELOG.md]: This was just an acceptance test code "typo" fix. Probably not needed in the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 ### Removed
 ### Fixed
+- Subadmin shouldn't be able to add users to their groups via API - [#31337](https://github.com/owncloud/core/issues/31337)
+- Remove session token creation for dav requests during loadApps - [#31428](https://github.com/owncloud/core/issues/31428)
+- Fix overflowing public share names in the share panel - [#31369](https://github.com/owncloud/core/issues/31369)
+- Fix occ user:sync to sync quota from preferences after upgrade if backend provided no quota - [#31360](https://github.com/owncloud/core/issues/31360)
+- Fix for Redis dev editions - [#31282](https://github.com/owncloud/core/issues/31282)
+- Fix mail debug message recipient field - [#31227](https://github.com/owncloud/core/issues/31227)
+- Prevent infinite loop in case of error in "log" event handler - [#31247](https://github.com/owncloud/core/issues/31247)
+- Convert FileContentNotAllowedException into DAVForbiddenException to … - [#31260](https://github.com/owncloud/core/issues/31260)
```
Comment (phil-davis) [inline CHANGELOG.md]: "to ..." looks odd.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,9 +6,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 ## [Unreleased]
 ### Added
+- occ user:list now takes a list of attributes to display - [#31115](https://github.com/owncloud/core/issues/31115)
 ### Changed
+- Bump symfony to 3.4.8 and other pending minor bumps - [#31221](https://github.com/owncloud/core/issues/31221)
+- Bump karma from 2.0.0 to 2.0.2 in /build - [#31253](https://github.com/owncloud/core/issues/31253)
+- Bump karma-jasmine from 1.1.1 to 1.1.2 in /build - [#31378](https://github.com/owncloud/core/issues/31378)
+- Bump karma-coverage from 1.1.1 to 1.1.2 in /build - [#31380](https://github.com/owncloud/core/issues/31380)
+- Update jsdoc requirement to ~3.5.5 - [#30036](https://github.com/owncloud/core/issues/30036)
+- deprecate Jenkins for pullrequest testing - [#31215](https://github.com/owncloud/core/issues/31215)
```
Comment (patrickjahns) [inline CHANGELOG.md]: not relevant?

**Comment by PVince81** [comment]
Thanks for the review, PR adjusted

**Review by phil-davis — APPROVED** [review]
Looks good so far.

**Pull request merged**